Automattic/jetpack
# Site Migrations: Update CTA on import.php wp-admin screen

Fixes https://github.com/Automattic/wp-calypso/issues/90945

## Proposed changes:

* We're updating references to `/setup/import-focused` to go to `/setup/hosted-site-migration` to take advantage of Migrate Guru's relative stability and success rates. 
* This updates an instance on the `/wp-admin/import.php` screen for some WP.com sites.
* Related project thread: paYKcK-4BA-p2 

<img width="1512" alt="Screenshot 2024-05-21 at 11 07 14 AM" src="https://github.com/Automattic/jetpack/assets/2124984/bd9096db-58ab-4075-9c82-e159d086073a">

**Tracking**

The URL has a `?ref=wp-admin` ref for tracking purposes. AFAIK there are no associated Tracks events with clicking on the CTA. Since this is a wp-admin thing on an Atomic site I'm not sure we can add Tracks events...


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* I have only seen this CTA on Woo Express sites (trial or paid plan) so far, so that's what I'm testing with.
* Create a new site from `/setup/wooexpress`
* Pay for the full plan and go to Settings -> Hosting Config to create your login credentials
* Either update the plugin manually to this branch via rsync, or use the Jetpack Beta Tester plugin pointed to this branch (don't forget to SSH into the site to add the `define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );` constant to your `wp-config.php` file.
* Visit the site's `/wp-admin/import.php` screen
* The "Get started" CTA should point to `/setup/hosted-site-migration/site-migration-identify?siteSlug=siteSlug&ref=wp-admin`
